app/package.json

```diff
@@ -10,15 +10,16 @@
     "url": "https://github.com/jupyterlite/jupyterlite"
   },
   "scripts": {
-    "clean": "rimraf build lab/build retro/build",
+    "clean": "rimraf build console/build lab/build retro/build",
```

Comment:
maybe we need a `lerna run clean`

Reply:
Or `rimraf build **/build`?

Otherwise that would probably mean adding back the `clean` scripts in each of the apps.

Reply:
Done in https://github.com/jupyterlite/jupyterlite/pull/498/commits/880847fd9b750319457b25495148aa00bb989928